src/components/swap/PendingModalContent.tsx

```diff
@@ -52,6 +63,70 @@ const LoadingIndicator = styled(Loader)`
   position: absolute;
 `
 
+function CurrencyLoader({ currency }: { currency: Currency | undefined }) {
+  const theme = useTheme()
+  return (
+    <LogoContainer>
+      <LogoLayer>
+        <CurrencyLogo currency={currency} size="48px" />
+      </LogoLayer>
+      <LoadingIndicator stroke={theme.textTertiary} />
+    </LogoContainer>
+  )
+}
+
+const PinkCircle = styled(LogoContainer)`
+  display: flex;
+  height: 48px;
+  width: 48px;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({ theme }) => theme.userThemeColor};
+  z-index: 1;
+`
+
+function PaperIcon({ currency, loading }: { currency: Currency | undefined; loading: boolean }) {
+  const theme = useTheme()
+  return (
+    <LogoContainer>
+      <PinkCircle>
+        <PapersIcon />
+        <CurrencyLogo
+          currency={currency}
+          size="20px"
+          style={{
```

Comment:
create a new styled component that wraps `CurrencyLogo` with these values

Reply:
i do this in the [next PR](https://github.com/Uniswap/interface/pull/6590/) when i refactor these logos to have animations